src/chat/chat_panel.rs

```diff
@@ -122,6 +122,7 @@ live_design! {
         main_section = {
             body_section = {
                 bubble = {
+                    padding: {left: 0, bottom: 0, top: 0}
```

Comment:
Most of bubble spacing is removed for assistant messages. Although the right padding was assumed when placing the jump to bottom button, so that's preserved.